src/py/flwr/server/superlink/state/in_memory_state.py

```diff
@@ -43,6 +43,7 @@ def __init__(self) -> None:
         self.client_public_keys: Set[bytes] = set()
         self.server_public_key: Optional[bytes] = None
         self.server_private_key: Optional[bytes] = None
+        self.public_key_node_id_pairs: Dict[bytes, int] = {}
```

Comment:
Let's move these next to `node_ids` and group all lines related to `node_ids`

Reply:
Done.